src/content/configuration/extending-configurations.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Extends
+sort: 12
+contributors:
+  - burhanuday
+---
+
+## extends
+
+`string | string[]`
+
+<Badge text="webpack v5.82.0+" />
+<Badge text="webpack-cli v5.0.3+" />
+
+The `extends` property allows you to extend an existing configuration to use as base. It internally uses the `webpack-merge` package to merge the configurations and helps you to avoid duplicating configuration between multiple configurations.
+
+W> **This option is not supported via the Node API**: Extends will have no effect when using the Node API. webpack-cli is required to use this feature.
```

Comment:
```suggestion

The `extends` property allows you to extend an existing configuration to use as the base. It internally uses the `webpack-merge` package to merge the configurations and helps you to avoid duplicating configurations between multiple configurations.

W> **This option is not supported via the Node API**: Extends will have no effect when using the Node API. webpack-cli is required to use this feature.
```